README.md

```diff
@@ -111,6 +121,12 @@ not "drain" from the simulation.
       "ca_cert": "/path/ca.pem",
       "client_cert": "/path/client.pem",
       "client_key": "/path/client-key.pem"
+    },
+    { 
+      "id": "carol",
+      "base_url": "http://127.0.0.1:8286",
```

Comment:
In this PR (https://github.com/bitcoin-dev-project/sim-ln/pull/212), we removed the requirement that URLs start with `https://` so we could remove it from the example.

Reply:
`serde_address` adds `https://` and local Eclair has `http://`

Reply:
For consistency, I think it would be necessary to do something similar with `base_ulr` detecting if it contains `http://` or not.

Reply:
do you mean similar to `serde_address ` ?
seems like this parsing only complicates the things.

Reply:
@f3r10 
would this be sufficient to keep consistency?

```
        if s.starts_with("https://") || s.starts_with("http://") {
            Ok(s)
        } else {
            Ok(format!("https://{}", s))
        }
```

Reply:
done: https://github.com/bitcoin-dev-project/sim-ln/pull/225/commits/b1901308d10d289b20f9eb6136ef5d5321385d70

Reply:
> The idea is not to force users to use https or , as in this case, http in the configuration file.

yea, it makes sense. thank you for details!